src/app/near/VeNearOnboardingModal.tsx

```diff
@@ -47,6 +47,11 @@ const STAKING_POOL_MAPPING: Record<string, StakingPool> = {
     name: "MetaPool",
     description: "5.98% Voting Power Growth",
   },
+  [process.env.NEXT_PUBLIC_NEAR_RNEAR_TOKEN_CONTRACT_ID ?? ""]: {
+    stakingPoolId: process.env.NEXT_PUBLIC_NEAR_RNEAR_TOKEN_CONTRACT_ID ?? "",
+    name: "rNEAR",
+    description: "9.64% APY",
+  },
```

Comment:
I think this modal isn't being used anywhere and can also be deleted instead of being updated to add rNEAR